src/constants.ts

```diff
@@ -28,9 +28,9 @@ export const PUBLIC_NETWORKS: { [chainId: number]: { name: string; etherscan: st
     etherscan: "https://polygonscan.com/",
   },
   324: { name: "zksync", etherscan: "https://explorer.zksync.io/" },
-  8453: { name: "base", etherscan: "https://mainnet.basescan.org" },
+  8453: { name: "base", etherscan: "https://basescan.org" },
```

Comment:
There's no trailing `/` here, but fwiw I think it's a mistake to require it here - we should be inserting the `/` when we generate the link.

Reply:
Resolved